modules/examiner/ajax/validateExaminer.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Certification training: Checks if the answers given for training quiz are correct.
+ *
+ * PHP Version 5
+ *
+ * @category Behavioural
+ * @package  Loris
+ * @author   Tara Campbell <tara.campbell@mail.mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+set_include_path(get_include_path().":../project/libraries:../php/libraries:");
+ini_set('default_charset', 'utf-8');
+require_once __DIR__ . "/../../../vendor/autoload.php";
+
+$config = NDB_Config::singleton();
+
+$fullName = $_POST['addName'];
+$siteID   = $_POST['addSite'];
+
+// Check that a name and site was submitted
+if (empty($fullName) || empty($siteID)) {
+    print 1;
```

Comment:
These codes aren't very clear... they should be defined as a const with a meaningful name, or, better yet, use a more descriptive json object.